photo_mapper_webserver/authentication/security.py

```
@@ -56,105 +56,51 @@ def create_refresh_token(user):
-    new_access = create_access_token(user)
-    new_refresh = create_refresh_token(user)
-
-    return new_access, new_refresh
-
-def revoke_refresh_token(refresh_token: str):
-    payload = decode_refresh_token(refresh_token)
-    RefreshToken.objects.filter(jti=payload["jti"]).update(revoked=True)
+    try:
+        return User.objects.get(id=payload["sub"])
+    except User.DoesNotExist:
+        raise InvalidAccessTokenError()
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)